Makefile

```diff
@@ -29,6 +34,21 @@ tools: $(TOOLS)
 $(TOOLS): oci-%:
 	go build -ldflags "-X main.gitCommit=${COMMIT}" ./cmd/$@
 
+.PHONY: man install uninstall
+
+_man:
+	go-md2man -in "cmd/oci-unpack/oci-unpack.1.md" -out "oci-unpack.1"
+	go-md2man -in "cmd/oci-image-validate/oci-image-validate.1.md" -out "oci-image-validate.1"
+	go-md2man -in	"cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md" -out "oci-create-runtime-bundle.1"
+
+install: _man
```

Comment:
May be some one not want to install acquiescently that meas is just compile, and we should persist a target (`$tools`) for developer.